src/poker/CheckPriority.py

```diff
@@ -216,7 +216,7 @@ def value(use: List[str]):
         return [1e3+twopair[0]+twopair[1]*100,'twopair',[left],use]
         #tie  pass
     elif(onepair!=0): 
-        zz=use
+        zz=use.copy()
```

Comment:
What is zz ?

Reply:
new list that keep card in hands instead of use
because we have modified use by ,pop()